apps/src/code-studio/pd/application/facilitator1819/Section1AboutYou.jsx

```diff
@@ -134,7 +128,7 @@ export default class Section1AboutYou extends Facilitator1819FormComponent {
       formatErrors.alternateEmail = "Must be a valid email address";
     }
 
-    if (data.zipCode && !ZIP_CODE_REGEX.exec(data.zipCode)) {
+    if (data.zipCode && !isZipCode(data.zipCode)) {
```

Comment:
I think the first clause is unnecessary - the REGEX.test function handles undefined

Reply:
It's 2 checks. I only want that error message when *something* is entered, but it's invalid. If the field is just missing, it will fail from required fields and still highlight red.

Reply:
![image](https://user-images.githubusercontent.com/8405316/33146432-f778cdae-cf78-11e7-94a5-4c0c4bf2fb15.png)
and
![image](https://user-images.githubusercontent.com/8405316/33146443-0435e5a4-cf79-11e7-829b-e0c8db5076bb.png)